monitoring/prober/scd/test_operation_simple_heavy_traffic.py

```diff
@@ -0,0 +1,302 @@
+"""Basic Operation tests with hundreds of operations created in the same area:
+
+  - make sure operations do not exist with get or query
+  - create 100 operations sequentially, with each covers non-overlapping area that are close to others
+  - get by IDs
+  - search with earliest_time and latest_time
+  - mutate
+  - delete
+  - confirm deletion by get and query
+"""
+
+import datetime
+import json
+
+from . import common
+from .common import SCOPE_SC, SCOPE_CI, SCOPE_CM
+from ..infrastructure import default_scope
+
+
+def _load_op_ids():
+  with open('./scd/resources/op_ids_100.json', 'r') as f:
+    return json.load(f)
+
+
+BASE_URL = 'https://example.com/uss'
+OP_IDS = _load_op_ids()
+
+ovn_map = {}
+
+
+def test_ensure_clean_workspace(scd_session):
+  for op_id in OP_IDS:
+    resp = scd_session.get('/operation_references/{}'.format(op_id), scope=SCOPE_SC)
+    if resp.status_code == 200:
+      resp = scd_session.delete('/operation_references/{}'.format(op_id), scope=SCOPE_SC)
+      assert resp.status_code == 200, resp.content
+    elif resp.status_code == 404:
+      # As expected.
+      pass
+    else:
+      assert False, resp.content
+
+
+# Preconditions: None
+# Mutations: None
+@default_scope(SCOPE_SC)
+def test_ops_do_not_exist_get(scd_session):
+  for op_id in OP_IDS:
+    resp = scd_session.get('/operation_references/{}'.format(op_id))
+    assert resp.status_code == 404, resp.content
+
+
+# Preconditions: None
+# Mutations: None
+@default_scope(SCOPE_SC)
+def test_ops_do_not_exist_query(scd_session):
+  time_now = datetime.datetime.utcnow()
+  end_time = time_now + datetime.timedelta(hours=1)
+
+  resp = scd_session.post('/operation_references/query', json={
+    'area_of_interest': common.make_vol4(time_now, end_time, 0, 5000, common.make_circle(-56, 178, 12000))
+  }, scope=SCOPE_SC)
+  assert resp.status_code == 200, resp.content
+  found_ids = [op['id'] for op in resp.json().get('operation_references', [])]
+  assert not _intersection(OP_IDS, found_ids)
+
+  resp = scd_session.post('/operation_references/query', json={
```

Comment:
nit: I think we can remove this and the SCOPE_CM check -- they are there in the original file to make sure queries in general don't succeed with the wrong scope, but that check is already covered in the other file and doesn't need to be repeated here.  Ditto on other SCOPE_CI & SCOPE_CM 403 checks in other sections below.

Reply:
Makes sense, removed.